index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>The Static Web - Quiz</title>
+  <link rel="stylesheet" href="css/quiz.css">
+</head>
+<body>
+
+  <article class="card">
+    <header class="productTitle">
+      <h1>Yellow Dunk Cards</h1>
+    </header>
+
+    <section class="productSectionOne">
```

Comment:
**Pro tip:** Naming classes for sections based on their sequence in the HTML can lead to confusion. If your product owner or designer ever wants to change the order in which the sections appear, it would lead confusion for the next developer who came along.

Consider naming the classes to reflect the content that will be inside them.